pkg/maglev/maglev.go

```diff
@@ -83,10 +78,12 @@ func getPermutation(backends []string, m uint64, numCPU int) []uint64 {
 	if batchSize == 0 {
 		batchSize = bCount
 	}
+	wp := workerpool.New(numCPU)
 
 	for g := 0; g < bCount; g += batchSize {
-		wg.Add(1)
-		go func(from int) {
+		g := g
```

Comment:
maybe just call this `from` as that's what it's used as in the function below, and drop the `from := g` from below.